com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/FrameQueueItem.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using MLAPI.Security;
+using MLAPI.Serialization.Pooled;
+
+namespace MLAPI
+{
+    /// <summary>
+    /// FrameQueueItem
+    /// Container structure for RPCs written to the Queue Frame
+    /// Used for both Inbound and Outbound RPCs
+    /// NOTE: This could eventually become obsolete as other systems mature
```

Comment:
hmm, should we be more elaborate about this? maybe mentioning upcoming plans around new stream/reader/writer interfaces etc. (whatever we have in our minds — having said that, this is just a thought and optional to do so, I'm OK with this too)

Reply:
I am going to leave a note in the new RPCQueue revisited task to add additional information if there is time.